ext/opentelemetry-ext-opentracing-shim/src/opentelemetry/ext/opentracingshim/__init__.py

```diff
@@ -0,0 +1,232 @@
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from contextlib import contextmanager
+
+import opentracing
+
+from opentelemetry.ext.opentracingshim import util
+from opentelemetry.trace import Event
+
+
+def create_tracer(tracer, scope_manager=None):
+    return TracerWrapper(tracer, scope_manager)
+
+
+class SpanContextWrapper(opentracing.SpanContext):
+    def __init__(self, otel_context):
+        self._otel_context = otel_context
+
+    def unwrap(self):
+        """Returns the wrapped OpenTelemetry `SpanContext` object."""
+
+        return self._otel_context
+
+    @property
+    def baggage(self):
+        raise NotImplementedError
+        # TODO: Implement.
+
+
+class SpanWrapper(opentracing.Span):
+    def __init__(self, tracer, context, span):
+        super().__init__(tracer, context)
+        self._otel_span = span
+
+    def unwrap(self):
+        """Returns the wrapped OpenTelemetry `Span` object."""
+
+        return self._otel_span
+
+    def set_operation_name(self, operation_name):
+        self._otel_span.update_name(operation_name)
+
+        # Return self for call chaining.
+        return self
+
+    def finish(self, finish_time=None):
+        end_time = finish_time
+        if end_time is not None:
+            end_time = util.time_seconds_to_ns(finish_time)
+        self._otel_span.end(end_time=end_time)
+
+    def set_tag(self, key, value):
+        self._otel_span.set_attribute(key, value)
+
+        # Return self for call chaining.
+        return self
+
+    def log_kv(self, key_values, timestamp=None):
+        if timestamp is not None:
+            event_timestamp = util.time_seconds_to_ns(timestamp)
+        else:
+            event_timestamp = util.time_ns()
+
+        event_name = util.event_name_from_kv(key_values)
+        event = Event(event_name, event_timestamp, key_values)
+        self._otel_span.add_lazy_event(event)
+
+        # Return self for call chaining.
+        return self
+
+    def set_baggage_item(self, key, value):
+        raise NotImplementedError
+        # TODO: Implement.
+
+    def get_baggage_item(self, key):
+        raise NotImplementedError
+        # TODO: Implement.
+
+    # TODO: Verify calls to deprecated methods `log_event()` and `log()` on
+    # base class work properly (it's probably fine because both methods call
+    # `log_kv()`).
+
+
+class ScopeWrapper(opentracing.Scope):
+    def close(self):
+        self._span.finish()
+        # TODO: Set active span on OpenTelemetry tracer.
+        # https://github.com/open-telemetry/opentelemetry-python/issues/161#issuecomment-534136274
+
+
+class ScopeManagerWrapper(opentracing.ScopeManager):
```

Comment:
Same naming comment as above.

Reply:
See https://github.com/open-telemetry/opentelemetry-python/pull/211#discussion_r334613932. Resolving this to keep the discussion in one place.